History.md

```diff
@@ -1,3 +1,9 @@
+unreleased
+==========
```

Comment:
You'd want to have a blank line between the header and first bullet point.

Reply:
This is fix at 3e7c7ab9a19d7cb510755e7e75394d591ddc48a3